dali/operators/reader/coco_reader_op.cc

```diff
@@ -20,62 +20,71 @@ DALI_REGISTER_OPERATOR(COCOReader, COCOReader, CPU);
+    "Path to the directory with meta files that contain preprocessed COCO annotations.",
     std::string())
   .AddOptionalArg("annotations_file",
       R"code(List of paths to the JSON annotations files.)code",
       std::string())
   .AddOptionalArg("shuffle_after_epoch",
-      R"code(If true, reader shuffles whole dataset after each epoch.)code",
+      R"code(If set to True, the reader shuffles the entire  dataset after each epoch.)code",
       false)
   .AddArg("file_root",
-      R"code(Path to a directory containing data files.)code",
+      R"code(Path to a directory that contains the data files.)code",
       DALI_STRING)
   .AddOptionalArg("ltrb",
-      R"code(If true, bboxes are returned as [left, top, right, bottom], else [x, y, width, height].)code",
+      R"code(If set to True, bboxes are returned as [left, top, right, bottom].
+
+If set to False, the bboxes are returned as [x, y, width, height].)code",
       false)
   .AddOptionalArg("masks",
-      R"code(If true, segmentation masks are read and returned as polygons.
-Each mask can be one or more polygons. A polygon is a list of points (2 floats).
-For a given sample, the polygons are represented by two tensors:
+      R"code(If set to True, segmentation masks are read and returned as polygons,
+which is a list of points (2 floats).
+
+Each mask can be one or more polygons, and for a given sample, the polygons are represented by the
+following tensors:
 
-- `masks_meta` -> list of tuples (mask_idx, start_idx, end_idx)
-- `masks_coords`-> list of (x,y) coordinates
+- ``masks_meta`` -> list of tuples (mask_idx, start_idx, end_idx)
+- ``masks_coords``-> list of (x,y) coordinates
 
-One mask can have one or more `masks_meta` having the same `mask_idx`, which means that the mask for that given
-index consists of several polygons).
-`start_idx` indicates the index of the first coords in `masks_coords`.
-Currently skips objects with `iscrowd=1` annotations (RLE masks, not suitable for instance segmentation).)code",
+One mask can have one or more ``masks_meta`` values that have the same ``mask_idx``.
+This means that the mask for that given index consists of several polygons.
+``start_idx`` indicates the index of the first coordinates in ``masks_coords``.
+Currently objects with ``iscrowd=1`` annotations are skipped because RLE masks are not suitable
+for instance segmentation.)code",
       false)
   .AddOptionalArg("skip_empty",
       R"code(If true, reader will skip samples with no object instances in them)code",
       false)
   .AddOptionalArg("size_threshold",
-      R"code(If width or height of a bounding box representing an instance of an object is under this value,
-object will be skipped during reading. It is represented as absolute value.)code",
+      R"code(If the width or the height of a bounding box that represents an instance of an object
+is lower than this value, the object will be skipped during reading.\n
```

Comment:
```suggestion
is lower than this value, the object will be ignored.\n
```
Ignored or Skipped, but let's drop the "during reading". It's just confusing.
Also, consider replacing \n with an empty line

Reply:
Done